ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -1,368 +1,373 @@
-        )
-    return ivy.split(
-        input, num_or_size_splits=indices_or_sections, axis=dim, with_remainder=True
-    )
-
-
-@to_ivy_arrays_and_back
-def tile(input, dims):
-    try:
-        tup = tuple(dims)
-    except TypeError:
-        tup = (dims,)
-    d = len(tup)
-    res = 0
-    if len(input.shape) > len([dims]) - 1:
-        res = input
-    if d < input.ndim:
-        tup = (1,) * (input.ndim - d) + tup
-        res = ivy.tile(input, tup)
-
-    else:
-        res = ivy.tile(input, repeats=dims, out=None)
-    return res
-
-
-@to_ivy_arrays_and_back
-def transpose(input, dim0, dim1):
-    return ivy.swapaxes(input, dim0, dim1)
-
-
-@to_ivy_arrays_and_back
-def unbind(input, dim=0):
-    shape = list(input.shape)
-    shape.pop(dim)
-    return tuple([x.reshape(tuple(shape)) for x in split(input, 1, dim=dim)])
-
-
-@to_ivy_arrays_and_back
-def unsqueeze(input, dim=0):
-    return ivy.expand_dims(input, axis=dim)
-
-
-@to_ivy_arrays_and_back
-def vsplit(input, indices_or_sections=None, /):
-    if isinstance(indices_or_sections, (list, tuple, ivy.Array)):
-        indices_or_sections = (
-            ivy.diff(indices_or_sections, prepend=[0], append=[input.shape[0]])
-            .astype(ivy.int8)
-            .to_list()
-        )
-    return tuple(ivy.vsplit(input, indices_or_sections))
-
-
-@to_ivy_arrays_and_back
-def vstack(tensors, *, out=None):
-    return ivy.vstack(tensors, out=out)
-
-
-@to_ivy_arrays_and_back
-def where(condition, input=None, other=None):
-    if not ivy.exists(input) and not ivy.exists(other):
-        return nonzero(condition, as_tuple=True)
-    return ivy.where(condition, input, other)
+# local
+import ivy
+from ivy.functional.frontends.torch.func_wrapper import (
+    to_ivy_arrays_and_back,
+    numpy_to_torch_style_args,
+    to_ivy_shape,
+)
+
+
+@to_ivy_arrays_and_back
+def adjoint(input):
+    return ivy.adjoint(input)
+
+
+@to_ivy_arrays_and_back
+def argwhere(input):
+    return ivy.argwhere(input)
+
+
+@to_ivy_arrays_and_back
+def cat(tensors, dim=0, *, out=None):
+    return ivy.concat(tensors, axis=dim, out=out)
+
+
+@to_ivy_arrays_and_back
+def chunk(input, chunks, dim=0):
+    if ivy.shape(input) == ():
+        return [input]
+    else:
+        dim_size = ivy.shape(input)[dim]
+        chunk_size = dim_size // chunks
+        if chunk_size == 0:
+            return ivy.split(input, num_or_size_splits=dim_size, axis=dim)
+        else:
+            remainder = dim_size % chunks
+            if remainder == 0:
+                return ivy.split(input, num_or_size_splits=chunks, axis=dim)
+            else:
+                return ivy.split(
+                    input,
+                    num_or_size_splits=tuple(
+                        [chunk_size + remainder] + [chunk_size] * (chunks - 1)
+                    ),
+                    axis=dim,
+                )
+
+
+@to_ivy_arrays_and_back
+def column_stack(tensors, *, out=None):
+    return ivy.stack(tensors, axis=1, out=out)

```

Comment:
according to the documentation:

```Creates a new tensor by horizontally stacking the tensors in tensors.

Equivalent to torch.hstack(tensors), except each zero or one dimensional tensor t in tensors is first reshaped into a (t.numel(), 1) column before being stacked horizontally.
```
could you please implement the `each zero or one dimensional tensor t in tensors is first reshaped into a (t.numel(), 1) column before being stacked horizontally` part please?

Reply:
Okay.

Reply:
thanks a lot @Purity-E